### ISSUE TITLE
Add a curated list of node to open channel

### ISSUE THREAD

**Issue by hsjoberg** [hsjoberg/blixt-wallet#1137]
Could be based on community nodes #1033

**Comment by Darth-Coin** [comment]
Yes this is necessary, especially for new Blixt users that don't know to which node should open. That list I created in #1033 is good but not so many new Blixt users know about it (even that is mentioned in many guides and FAQ - people don't read).

Even a link (inside Blixt) to that list is enough for the moment until a better solution will be found.

**Comment by hsjoberg** [comment]
I'm not sure exactly where we should keep this list.

We could keep track of it in a JSON file hosted on blixtwallet.github.io.
Or we can hardcode it inside Blixt Wallet itself.

Having outside Blixt makes it easy to update it.

**Comment by hsjoberg** [comment]
Related issue #1187

**Comment by Darth-Coin** [comment]
We can extend this list also with the nodes that offer neutrino servers.
Sometimes help users with stuck sync, just by switching to another neutrino. Who knows maybe is about caching IPs or something. Happen to me once.

But we need this list in Blixt ASAP. People have no idea with who to open channels.
I will try to add more nodes, but I need to vouch them first. Not all are good.
